File: ClientSide/UkrainianStyleRestaurant_2.0_another/app/src/main/java/com/example/ukrainianstylerestaurant/model/DeliveryStatusResponse.java
package com.example.ukrainianstylerestaurant.model;

public class DeliveryStatusResponse {
    public int id;
    public int status; // 0=Created, 1=Assigned, 2=PickedUp, 3=Delivered
    public Integer courierId; // Integer, щоб могло бути null
    public String deliveredAt; // Можна додати, якщо треба
}

File: API/DeliveryService/Models/DeliveryStatusResponse.cs
using DeliveryService.Entities;

namespace DeliveryService.Models
{
    public class DeliveryStatusResponse
    {
        public int Id { get; set; }
        public DeliveryStatus Status { get; set; }
        public int? CourierId { get; set; }
        public DateTime? DeliveredAt { get; set; }
    }
}